v3/fleet/control-jenkins.service

```diff
@@ -11,7 +11,9 @@ Restart=always
 RestartSec=4
 TimeoutStartSec=0
 ExecStartPre=-/usr/bin/sudo rm -rf /home/core/control-jenkins
-ExecStartPre=/usr/bin/git clone https://github.com/adobe-platform/control-jenkins.git /home/core/control-jenkins
+# ExecStartPre=/usr/bin/git clone https://github.com/adobe-platform/control-jenkins.git /home/core/control-jenkins
+ExecStartPre=/usr/bin/git clone --branch okta --single-branch https://github.com/remusb/control-jenkins.git /home/core/control-jenkins
```

Comment:
We can't publish this as-is (using your custom github link). It will have to be merged into master or versioned somehow.